Ensure `fio` uses full unix timestamps

portante opened the pull request:
Addresses issuse #913 and fixes issue #1021.

Note that the generated HTML for `tests/pbench-fio/test-32.txt` changes from using an `xy` chart to a `timeseries` chart because the existing code at [line 73](https://github.com/distributed-system-analysis/pbench/blob/main/agent/bench-scripts/postprocess/fio-postprocess-viz.py#L73) of `fio-postprocess-viz.py` uses `timeseries` when it sees `log_unix_epoch` in the `fio.job` file.

Note that our tests do not run `fio` so we only ensure the option is now showing up in `fio.job` files.

dbutenhof reviewed (commented):
The git comment doesn't explain the change from "xy" to "timeseries" charting; should it?

And neither change has any impact on the legacy test output other than echoing the options; which is slightly concerning as we're getting no proof that these are accomplishing anything. I'm a little surprised there are no other file diffs detected.

webbnh reviewed (changes requested):
I second Dave's comments:  I don't see any code (or configuration) change which would have prompted the change from "xy" plot to "timeseries" in `test-32.txt`.  And, I don't see anything in this PR which indicates that it addressed the problem reported in #1021.

portante commented:
> The git comment doesn't explain the change from "xy" to "timeseries" charting; should it?

I updated the git commit message and the PR description to cover this.

> And neither change has any impact on the legacy test output other than echoing the options; which is slightly concerning as we're getting no proof that these are accomplishing anything. I'm a little surprised there are no other file diffs detected.

We don't run `fio` as part of the legacy tests, so we only update the `fio.job` file to ensure it is present.  Also updated the git commit message and the PR description.

portante commented:
> I second Dave's comments: I don't see any code (or configuration) change which would have prompted the change from "xy" plot to "timeseries" in `test-32.txt`.

Answered in the git commit message and PR description updates.

> And, I don't see anything in this PR which indicates that it addressed the problem reported in #1021.

Due to the fact that we don't run `fio` as part of the existing legacy tests, we can't demonstrate that at this time, other than to ensure the proper option is present in the generated `fio.job` files.

webbnh reviewed (approved):
👍

portante reviewed (commented):
Had to rebase.

webbnh reviewed (approved):
> Had to rebase.

You didn't want to use the "Rebase and merge" button?  (Or, even the "Squash and merge" button?)
